keep/api/models/db/action.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+from typing import Optional
+
+from sqlalchemy import UniqueConstraint
+from sqlmodel import Column, Field, SQLModel, TEXT
+
+
+class Action(SQLModel, table=True):
+    __table_args__ = (UniqueConstraint("tenant_id", "name", "use"),)
+
+    id: str = Field(default=None, primary_key=True)
+    tenant_id: str = Field(foreign_key="tenant.id")
+    use: str
```

Comment:
what's the difference between `use` and `name`?

Reply:
About the `use` and `name` fields on the model:

- `use` is the key to find the corresponding action template to be expanded from `actions` section into `alert`/ `workflow` section.
- `name` is just a normal name of action defined in the template. I plan to save it as a separate field for displaying purpose (just in case we support configuration of reusable actions on the UI later). 

```yaml
actions:
  - name: trigger-slack2
    use: '@trigger-slack2-use-id'
    provider:
      type: slack
      config: " {{ providers.slack-demo }} "
      with:
      ...
  - name: trigger-slack3
    use: '@trigger-slack3-use-id'
    provider:
      type: slack
      config: " {{ providers.slack-demo }} "
      with:
      ...
```